### PR TITLE
Add PostgreSQL Paglia sample database howto

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Hi there!

This PR adds the PostgreSQL Paglia sample database as a HowTo article. It explains a bit about the database, how to import it (coming soon!), its ER model, some sample queries, license etc.

I also included the SQL source files in the repository, since they are quite small and we can reference them easily here.

Let me know your feedback!

### CODE REVIEW

**Comment by jonatasbaldin** [comment]
On Vale: got [this](https://github.com/aiven/devportal/runs/3250332040?check_suite_focus=true) error, but the string is actually a link, so it should keep as `postgres` instead of `PostgreSQL`. Is there a way to make an exception? I've seem other instances of `www.postgresql.org` that are not triggering the error 🤔

**Comment by lornajane** [comment]
I think this is happening because of the image on line 24 which is missing an alt tag (I realise this isn't a good error for that situation, but images should have alt tags and also this error will go away)

**Comment by jonatasbaldin** [comment]
Flagging this a draft. I'll add more information on how to load the data / connect to the database once the feature is implemented.

**Comment by lornajane** [comment]
I'd like to re-iterate that I think we should publish this as soon as we possibly can, explaining how a user can perform the steps themselves without any Aiven-specific features.

**Comment by jonatasbaldin** [comment]
@lornajane got it, will get back to this :)

**Comment by jonatasbaldin** [comment]
hey @lornajane, I added more instructions to actually load the database and use it, can you take a look now?

**Comment by jonatasbaldin** [comment]
@lornajane could you help around with the latest build failure? those terms are on `accept.txt` file 🤔

File: .github/styles/Aiven/capitalization_headings.yml
Comment (lornajane) [inline .github/styles/Aiven/capitalization_headings.yml:9]: What is this doing here? The exceptions are needed for words that cannot be added to the dictionary, but don't meet the sentence case rules (HowTo has a second uppercase letter so it causes a problem).

File: .github/styles/Vocab/Docs/accept.txt
Comment (lornajane) [inline .github/styles/Vocab/Docs/accept.txt:7]: Where did we need to add this? Spelling check shouldn't hit things with the `literal` markup

File: .github/styles/Vocab/Docs/accept.txt
Comment (lornajane) [inline .github/styles/Vocab/Docs/accept.txt:8]: Is this a word ....?

**Review by lornajane — CHANGES_REQUESTED** [review]
I put a couple of comments in line but this is a very nice starting point for a new postgresql user and I like the example queries. I'm less sure about showing the output in code view, let's get some other opinions on how to present this information. I've got some minor comments and questions, mostly inline.

Where you link to other locations, please make the links some descriptive words of what is on the other end of the link. If it's just "here", then it's very difficult for tools to, for example, index links (and a screenreader will simply give a choice of clicking here, here, here, or here - which is as helpful as it sounds).

**Comment by lornajane** [comment]
When I imported the pagila database into a standard startup-4 database, I get the following (including the last N lines of output in case that helps to diagnose)

```
ALTER TABLE
psql:pagila-data.sql:48079: ERROR:  insert or update on table "customer" violates foreign key constraint "customer_address_id_fkey"
DETAIL:  Key (address_id)=(88) is not present in table "address".
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
ALTER TABLE
pagila=> 
```

**Comment by hab-anka** [comment]
I wonder whether this should sit in **Integrations** or somewhere else? Also maybe we can revisit the title? I think this may be technically an integration, but it's all about _Using sample data_, or _Testing the database_.

**Comment by jonatasbaldin** [comment]
Thanks @lornajane for the review and changes.

About the name, it is `Pagila` but I created the article with the misspelled `Paglia` and corrected it later, sorry for the confusion. Also, the dropdown looks great.

@hab-anka I was wondering about creating a new section, but I don't know what it should be called. Something like `Sample data` or `Import data` (?). On the title, I don't mind change, I think we need to figure it out where the content will live first and them adjust accordingly. What do you think?

**Comment by jonatasbaldin** [comment]
@lornajane I'll check the error you mentioned later.

File: docs/products/postgresql/howto/pagila.rst
Comment (hab-anka) [inline docs/products/postgresql/howto/pagila.rst:15]: We can try to just say that you need a running PostgreSQL service.

File: docs/products/postgresql/howto/pagila.rst
Comment (hab-anka) [inline docs/products/postgresql/howto/pagila.rst:55]: Is this part of the generic info for this sample data or is it how we've set it up? If it's specific to Aiven, I would move this up to where the conceptual info sits (what is Pagila etc.), and maybe even think of separating the conceptual info from the actual task.

File: docs/products/postgresql/howto/pagila.rst
Comment (hab-anka) [inline docs/products/postgresql/howto/pagila.rst:224]: I wonder if we can give actual answers (hidden somehow) to these challenges - as this is just hanging here as an open question :) We can use the twistee section (thx, @lornajane) at the end of the page?

File: docs/products/postgresql/howto/pagila.rst
Comment (hab-anka) [inline docs/products/postgresql/howto/pagila.rst:247]: I think we can skip it also by linking to this info.

**Review by hab-anka — COMMENTED** [review]
I like how this looks like! Some bigger questions to answer about how we treat this type of super useful content. And some smaller suggestions about removing and moving things around. Happy to look at the actual text soon!

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:1]: Keeping this since Pagila is a known data sample out there, so if users spot the name, they might know what this is about.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:4]: I kept the link on the original place since it makes sense to talk about it a bit before linking to it. On the first paragraph, I just mention what the article is about.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:17]: Agreed, linked to the new page.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:15]: Ajusted.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:224]: I was thinking about a more open challenge without answers, so people can discover themselves.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:55]: This is info for the sample data, not related to Aiven itself, it only makes sense on this article.

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:1]: ✅

File: .github/styles/Vocab/Docs/accept.txt
Comment (jonatasbaldin) [reply .github/styles/Vocab/Docs/accept.txt:51]: ✅

File: _toc.yml
Comment (jonatasbaldin) [reply _toc.yml:73]: ✅

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:261]: ✅

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:263]: ✅

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:17]: It's live now ✨

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:30]: As soon as we merge, I'll push a new PR to fix this link. I did like this so people could test the whole article and because the CI was complaining about missing links heh

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:50]: ✅

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:62]: ✅

File: docs/products/postgresql/howto/pagila.rst
Comment (ftisiot) [inline docs/products/postgresql/howto/pagila.rst:16]: Should we add a couple of subtitles here? 
e.g. "Load Sample Dataset via button" vs "Load Sample Dataset manually"

When I initially read the article I got lost into thinking I had to do both.

File: docs/products/postgresql/howto/pagila.rst
Comment (ftisiot) [inline docs/products/postgresql/howto/pagila.rst:240]: Should we also give the solutions on a different page? I feel this could be like the crosswords, where you know you can find the solution at the end of the book

File: docs/products/postgresql/howto/pagila.rst
Comment (ftisiot) [inline docs/products/postgresql/howto/pagila.rst:67]: Should we add a bit more wording here? The picture is nice, but an unexperienced user might struggle to find the information needed. 
Something like "the actors table contains ...., and can be joined with the film table via the `film_actor` table", we don't need to be exaustive, but just giving the end reader a bit more knowledge on why we're building the queries like we do.

I often see the struggle in stackoverflow being in understanding the datamodel, before even starting with sql

**Review by ftisiot — COMMENTED** [review]
Lovely job! I left a couple of minor comments that I believe could make the whole dataset experience easier to read and understand.

Good job @jonatasbaldin

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:16]: Done!

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:243]: Done!

File: docs/products/postgresql/howto/pagila.rst
Comment (jonatasbaldin) [reply docs/products/postgresql/howto/pagila.rst:68]: Done!

**Review by lornajane — APPROVED** [review]
Approved BUT assigning this to you Jojo since it isn't well architected for the two-step deploy that this PR will need. Next steps as I see them are:
 - merge
 - open new PR with link to new URL for dataset
 - once that has been reviewed and merged, delete this branch

I'm therefore assigning this PR to you to handle (please let me know if you need a hand to rebase and merge, since it's been open a while. Please do not merge main into the feature branch.)

**Comment by jonatasbaldin** [comment]
Well, got slapped by the rebase 😤 will fix it shortly, pls don't merge 😓

**Pull request merged**
